tests/packages/media-sound/bitwig-studio-1.3.16.conf

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cat >/etc/portage/package.use/bitwig-studio <<EOF
```

Comment:
Can you change this into a simple `echo "media-libs/mesa -llvm" > /etc/portage/package.use/bitwig-studio`?

Reply:
@simonvanderveldt ping

Reply:
@mrVanDalo Sorry, was on holidays :)

> I must ask, is THIS really a blocker ?
> I mean we are talking about code style here, on a level that is quite detailed.

IMHO it is, because coding style only works if it's used consistently.
The shell script style guide says to use heredocs in case the line longer than 80 characters. It isn't in this case, so we should just use the simpler form.

Reply:
> coding style only works if it's used consistently.
I would disagree. coding style is still a thing left to the developer. Following blindly rules which pop up to quickly, will not result in good code quality.

> The shell script style guide says to use heredocs in case the line longer than 80 characters. It > isn't in this case, so we should just use the simpler form.
As you point out the docs talk about lines which are longer than 80 chars. But it says nothing about lines which are shorter.

I still don't see a point to use `echo` here. It will not improve readability.

Reply:
Quote from the styleguide, emphasis mine
> **If** you have to write strings that are longer than 80 characters, this should be done with a here document or an embedded newline if possible.

So using heredocs is only suggested in case you have more than 80 characters on a line. In other words, not for cases with less than 80 characters on a line.

Reply:
> In other words, not for cases with less than 80 characters on a line.

You can't reverse the sentence for the opposite situation.
"When it's raining it's getting cold" This does not imply "When it's not raining it's not getting cold"

Reply:
@marcan It would be nice if you could also join the discussion, so we can achieve some kind of democratic consents here.

Or we could make a quick discussion about style-guides and such in the irc.